### DISCUSSION TITLE
Running storybook with venia-ui Field and TextInput, I get a formApi.getValue error

### DISCUSSION THREAD

**Discussion by MichaelHeinzman** [magento/pwa-studio#4267] [Q&A]
I'm utilizing a Form from the react package Informed. I'm then importing Field and TextInput into the react application. Everything works when I wrap the Field and TextInput in the Form. However, when running storybook, I get the error formApi.getValue is not a function. I'm not sure if there's any documentation on how to handle this with the venia-ui package. It seems like the context from the Form isn't working when running storybook.

If anyone has any solutions it would be much appreciated.

**Comment by brendanfalkowski** [comment]
We don't use Storybook, but Informed docs aren't the clearest about explaining nested components vs single-component forms as it relates to the API usage. It's a bit annoying that you can't just import the hooks and it depends on how the components are written.

This may help: https://github.com/teslamotors/informed/issues/261

**Reply by MichaelHeinzman** [reply]
Thanks for the quick response, I guess I just won't be using storybook on those files lol.
